pom.xml

```diff
@@ -2678,7 +2678,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-enforcer-plugin</artifactId>
-                <version>1.4.1</version>
```

Comment:
why ?

Reply:
already defined in parent org.apache:apache